docs/_frontend/src/models/App/App.js

```diff
@@ -64,7 +68,7 @@ const fromSearchString = () => {
         break;
 
       case 'supported':
-        if (params[key] === '1') {
+        if (searchParams.get(key) === '1') {
```

Comment:
During the test, I noticed this bug that caused the support checkbox doesn't update from URL params so I fixed it,